test_scripts/API/VehicleData/GenericNetworkSignalData/036_Subscribtion_after_removing_VD_from_permissions.lua

```diff
@@ -35,13 +35,14 @@ common.writeCustomDataToGeneralArray(common.customDataTypeSample)
 common.setDefaultValuesForCustomData()
 
 local appSessionId = 1
-local itemToRemove = "custom_vd_item1_integer"
+local itemToRemoveName = "custom_vd_item1_integer"
+local itemToRemoveKey =  common.VehicleDataItemsWithData[itemToRemoveName].key
```

Comment:
@GetmanetsIrina Since `itemToRemoveKey` is used only in `expectFunc()` function I would suggest to move this variable into this function.
Also in this case `itemToRemoveName` could be reverted back to `itemToRemove`

Reply:
@dboltovskyi, updated in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2281/commits/6c4486456bf2b9903cc5ba8261b0bd541fadcd8e